docs/pbc/all/merchant-management/202407.0/marketplace/marketplace-merchant-commission-feature-overview.md

```diff
@@ -0,0 +1,191 @@
+---
+title: Marketplace Merchant Commission feature overview
+last_updated: Apr 23, 2021
+description: Merchant categories help you easily find relevant merchants.
+template: concept-topic-template
+---
+
+
+The Marketplace Commissions enables marketplace owners to monetize their services by calculating a fee on sales made through their platform. This feature allows for flexible commission structures, such as fixed fees, percentage-based fees, or a combination of both. Commissions can apply differently across various categories, products, or sellers. For example, you might want to collect a smaller commission from merchants that sell products in categories with smaller selling volumes.
+
+## Managing merchant commissions
+
+A Back Office user can import commissions by following [Import merchant commissions in the Back Office](/docs/pbc/all/merchant-management/202407.0/marketplace/import-merchant-commissions-in-the-back-office.html).
```

Comment:
We need to add the following info here:

We recommend using [Google Sheets](https://workspace.google.com/products/sheets/), which is well-suited for formatting CSV files.

The CSV file for the merchant commission import contains commission-related data you can import into your system. Check out the [template](https://spryker.s3.eu-central-1.amazonaws.com/docs/Developer+Guide/Back-End/Data+Manipulation/Data+Ingestion/Data+Import/Importing+Product+Data+With+a+Single+File/TEMPLATE+Product+import+with+single+file.csv) of the file for details on the data it contains.

Also mention that commissions cam be exported from Backoffice UI, modified and imported back using Google Sheets.